rust-packages/ic-verifiable-credentials/src/lib.rs

```diff
@@ -624,10 +624,18 @@ fn inconsistent_jwt_claims(custom_message: &'static str) -> JwtValidationError {
     ))
 }
 
-/// Extracts and returns raw canister sig public key (without DER-prefix) from the given header.
+/// Extracts and returns the raw canister sig public key (without DER-prefix) from the given header.
 pub fn get_canister_sig_pk_raw(
     jws_header: &JwsHeader,
 ) -> Result<Vec<u8>, SignatureVerificationError> {
```

Comment:
Could we use two different types for the raw and der public keys?

Reply:
We could. But ideally these types would be provided by some canister sig library rather than the VC SDK.

I'll talk to the crypto team to make this consistent, as the problem is already introduced [here](https://github.com/dfinity/ic/blob/211fa36d1de895d0295ecb27aa174756e881237b/packages/ic-signature-verification/src/canister_sig.rs#L16).

I'd suggest making this a separate PR, wdyt?

Reply:
Sounds good.